presto-main/src/main/java/com/facebook/presto/operator/scalar/TDigestFunctions.java

```diff
@@ -136,4 +137,37 @@ public static Block destructureTDigest(@SqlType("tdigest(double)") Slice input)
         blockBuilder.closeEntry();
         return TDIGEST_CENTROIDS_ROW_TYPE.getObject(blockBuilder, blockBuilder.getPositionCount() - 1);
     }
+
+    @ScalarFunction(value = "construct_tdigest", visibility = EXPERIMENTAL)
+    @Description("Create a TDigest by passing in its internal state.")
+    @SqlType("tdigest(double)")
+    public static Slice constructTDigest(
+            @SqlType("array(double)") Block centroidMeansBlock,
```

Comment:
should we call this means?

Reply:
sure, I can update these parameters to be `means` and `weights` respectively